IdentityCore/src/oauth2/token/MSIDPrimaryRefreshToken.m

```diff
@@ -40,7 +40,7 @@ - (instancetype)initWithTokenCacheItem:(MSIDCredentialCacheItem *)tokenCacheItem
         
         if (!_sessionKey)
         {
-            MSID_LOG_ERROR(nil, @"Trying to initialize primary refresh token when missing session key field");
+            MSID_LOG_WARN(nil, @"Trying to initialize primary refresh token when missing session key field");
```

Comment:
Isn't this a critical error?

Reply:
Corrupted items in keychain are skipped and considered non critical.